index.html

```diff
@@ -5,157 +5,190 @@
+                </div>
             </nav>
         </header>
 
 
         <main class="main1">
             <section class="section1">
-                <div class="gap">gap</div>
-                <div class="intro">
-                    <h1>Hey There. <br>
-                        I'm Usama
-                    </h1>
-                </div>
-                <div class="profession"><h2>I'am a Software Developer</h2></div>
-                <div class="containe">
-                    <div class="containe1">
-                        <div>
-                            <article class="services">
-                                <p>
-                                    I can help you build a product , feature or <br>
-                                    website Look through some of my work <br>
-                                    and experience! If you like what you see <br>
-                                    and have a project you need coded, <br>
-                                    don’t hestiate to contact me. <br>
-                                </p>
-                            </article>
-                            <ul class="logo">
-                                <li><a href="https://github.com/UsamaMasood12"><img src="./image/G.png" alt=""></a></li>
-                                <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./image/Group.png" alt=""></a></li>
-                                <li><a href="#"><img src="./image/M.png" alt=""></a></li>
-                                <li><a href="https://twitter.com/Usama____Masood"><img src="./image/twitter.png" alt=""></a></li>
-                                <li><a href="#"><img src="./image/V.png" alt=""></a></li>
-                            </ul>
+                <div class="aaa">
+                    <div class="aaaa">
+                        <div class="intro">
+                            <div class="gap"></div>
+                            <h1>Hey There. <br>
+                                I'm Usama
+                            </h1>
                         </div>
+                        <div class="profession"><h2>I'am a Software Developer</h2></div>
                     </div>
-                    <div class="containe2">
-                        <div class="box box1"><img src="./image/44.png" alt=""></div>
+                    <div class="containe">
+                        <div class="containe1">
+                            <div class="aaaaa">
+                                <article class="services">
+                                    <p>
+                                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                                        don’t hestiate to contact me.
+                                    </p>
+                                </article>
```

Comment:
- [ ] Great job 👍 You gave this text a `font-family` of `Poppins` instead of `Inter`. Kindly fix this 🙏